apps/src/code-studio/assets/DefaultSpriteRow.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+
+export default class DefaultSpriteRow extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    keyValue: PropTypes.string.isRequired,
+    onDelete: PropTypes.func.isRequired
+  };
+
+  render() {
+    return (
+      <div style={styles.assetRow}>
+        <Button
+          text={'Delete'}
```

Comment:
nit: string attribute values can be expressed directly as strings like `text="Delete"` (there are a couple instances of this below as well)